.github/workflows/node.js.yml

```diff
@@ -32,6 +32,9 @@ jobs:
       run: npm install
     - name: Build
       run: npm run build
+    - name: Lint
+      if: ${{ matrix.node-version == '10.16' }}
+      run: npm run lint
```

Comment:
Why only linting that one version?

Reply:
The linting fails for the `6.x` and `8.x` versions because `eslint` no longer supports those versions. Since the result of the `lint` will be the same regardless of version, I went with only running on one version that I know it will work with, rather than excluding multiple versions where it won't work.